moment/moment-node.d.ts

```diff
@@ -115,6 +115,18 @@ declare module moment {
         toJSON(): string;
     }
 
+    interface MomentLocale {
+        ordinal(n: number): string;
+    }
+
+    interface MomentCreationData {
+        input?: string,
+        format?: string,
+        locale: MomentLocale,
+        isUTC: boolean,
```

Comment:
Actually I didn't know.
This is what I've concluded after testing it for a little while.